### PR TITLE
Fix/ux papercuts account details issue #12292

### PR DESCRIPTION
**Pull request by igorms-cons**
Fixes: 
Updating account details modal view
(new branch for CLA issues)

Issue: https://github.com/MetaMask/metamask-extension/issues/12292

### CODE REVIEW

**Review by david0xd — DISMISSED** [review]
Looks good. 
As agreed on the way with requirements related to the figma.
Fore more info consult: https://github.com/MetaMask/metamask-extension/issues/12292

**Review by david0xd — DISMISSED** [review]

**Review by darkwing — CHANGES_REQUESTED** [review]

**Review by danjm — COMMENTED** [review]
LGTM, just need to fix e2e tests

**Comment by igorms-cons** [comment]
> LGTM, just need to fix e2e tests

@danjm  fixed

**Review by danjm — DISMISSED** [review]

**Review by darkwing — DISMISSED** [review]

File: ui/components/ui/qr-code/qr-code.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useState } from 'react';
 import qrCode from 'qrcode-generator';
 import { connect } from 'react-redux';
 import { isHexPrefixed } from 'ethereumjs-util';
+import copyToClipboard from 'copy-to-clipboard';
```
Comment (adonesky1) [inline ui/components/ui/qr-code/qr-code.js:6]: we have a hook `useCopyToClipBoard` that we should probably use here instead?

**Review by adonesky1 — CHANGES_REQUESTED** [review]
On small thing!

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Comment by igorms-cons** [comment]
@danjm  @adonesky1  all good, thx!

**Review by darkwing — APPROVED** [review]

**Review by danjm — APPROVED** [review]

**Pull request merged**